From: intrigeri <intrigeri@boum.org>
Date: Wed, 12 Aug 2015 09:21:28 +0000
Forwarded: not-needed
Bug: https://github.com/micahflee/torbrowser-launcher/issues/181
Subject: Set torbrowser.start-tor-browser and usr.bin.torbrowser-launcher
 AppArmor profiles to complain mode.

They have been broken for three months now. Patches have been discussed on
GitHub for a while, but no clean solution has been implemented yet,
so for the time being, let's make torbrowser-launcher *work*.
---
 apparmor/usr.bin.torbrowser-launcher  | 2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/apparmor/usr.bin.torbrowser-launcher b/apparmor/usr.bin.torbrowser-launcher
index b8e081e..80aace4 100644
--- a/apparmor/usr.bin.torbrowser-launcher
+++ b/apparmor/usr.bin.torbrowser-launcher
@@ -1,7 +1,7 @@
 # Last Modified: Thu Jan  2 15:12:38 2014
 #include <tunables/global>
 
-/usr/bin/torbrowser-launcher {
+/usr/bin/torbrowser-launcher flags=(complain) {
   #include <abstractions/base>
   #include <abstractions/nameservice>
   #include <abstractions/python>
